app/src/main/kotlin/com/wire/android/ui/authentication/create/details/DetailsScreen.kt

```diff
@@ -44,13 +47,14 @@ import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.launch
 
 @Composable
-fun DetailsScreen(viewModel: CreatePersonalAccountViewModel) {
+fun DetailsScreen(viewModel: CreatePersonalAccountViewModel, title: String) {
```

Comment:
why is this not inside the state object ?

Reply:
These screens can have different "title" (text shown on the `TopAppBar`) depending on the flow it's used in (personal account or team creation) so it's the parameter coming from outside the screen and not its inner state